TALOS-MIGRATION-GUIDE.md

```
@@ -0,0 +1,1121 @@
+# Save cluster secrets to 1Password (backup)
+op item create \
+  --vault "Homelab" \
+  --category "Secure Note" \
+  --title "talos-cluster-secrets" \
+  --note-value "$(cat talos/generated/secrets.yaml)"
+```
+
+### Step 2.3 — Create control-plane-1 VM on Proxmox Node 1
+
+**Option A: Using the installer script**
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)